pages/api.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: API
+permalink: /api/
+---
+
+The Pair Columbus API is a public, read-only resource for information related to the group and its resources. For now, it just serves static JSON used to generate the [Challenges page](/challenges/).
+
+## [Challenges](/api/challenges.json)
+This endpoint returns all active Pair Columbus challenges in categories (generally based on difficultly or technologies used).
+
+### Request
+`GET http://paircolumbus.org/api/challenges.json`
```

Comment:
It seems like this is a side effect of how curl handles caching, since Postman disables caching by default.